mig-blue-green/testing/test.cloudbuild.yaml

```diff
@@ -0,0 +1,84 @@
+substitutions:
+  _SANDBOX_PROJECT_NAME: undefined
+tags:
+  - "mig-blue-green-testing"
+steps:
+  - id: test-setup-sh
+    name: gcr.io/cloud-builders/gcloud
+    env:
+      - '_SANDBOX_PROJECT_NAME=${_SANDBOX_PROJECT_NAME}'
+    script: |
+      #!/bin/bash
+      set -e
+
+      echo "Running setup.sh in ${_SANDBOX_PROJECT_NAME}"
+      bash setup.sh ${_SANDBOX_PROJECT_NAME} yes
+
+  - id: test-apply-tf
+    name: gcr.io/cloud-builders/gcloud
+    env:
+      - '_SANDBOX_PROJECT_NAME=${_SANDBOX_PROJECT_NAME}'
+    script: |
+      #!/bin/bash
+      set -e
+
+      echo "Executing pipelines/apply.cloudbuild.yaml"
+      gcloud builds submit \
+          . \
```

Comment:
period should not be needed if using a config

Reply:
done